.github/workflows/phpunit-mysql.yml

```diff
@@ -8,11 +8,11 @@ on:
       - stable*
 
 env:
-  APP_NAME: notestutorial
+  APP_NAME: droneci_fast_lane
   # Location of the phpunit.xml and phpunit.integration.xml files
   # Comment to disable
   PHPUNIT_CONFIG: ./tests/phpunit.xml
-  PHPUNIT_INTEGRATION_CONFIG: ./tests/phpunit.integration.xml
+  #PHPUNIT_INTEGRATION_CONFIG: ./tests/phpunit.integration.xml
```

Comment:
should not be needed in the newest template. They perform a file exists check and otherwise skip